pennylane/devices/default_qubit_autograd.py

```diff
@@ -34,6 +37,9 @@ class DefaultQubitAutograd(DefaultQubitLegacy):
 
         pip install autograd
 
+    .. warning::
+        This device is deprecated. Use :class:`~.devices.DefaultQubit` instead; for example through ``qml.device("default.qubit")``, which now supports backpropagation.
```

Comment:
```suggestion
        This device is deprecated. Use :class:`~pennylane.devices.DefaultQubit` instead; for example through ``qml.device("default.qubit")``, which now supports backpropagation.
```

Can you make the same change for the other devices too?